### PR TITLE
Normalizing Public Cloud documentation

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Bringing documents up to the agreed standards of writing. 

# Target branches

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue [#<insert spacewalk issue, if any>](https://github.com/SUSE/spacewalk/issues/23122)
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/specialized-guides/pages/public-cloud-guide/payg/payg-limitations.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
-= {payg} limitations
+= {payg} Limitations
 
-== Onboarded {byos} instances without SCC credentials
+== Onboarded {byos} Instances without SCC Wredentials
```
Comment (jcayouette) [inline modules/specialized-guides/pages/public-cloud-guide/payg/payg-limitations.adoc:3]: Credentials

File: modules/specialized-guides/pages/public-cloud-guide/payg-support.adoc [hunk 2]
```diff
@@ -48,24 +55,28 @@ Limits the features to check.
...
 
-== Billing services and troubleshooting
+
+== Billing Services and Troubleshooting
 
 
 === billing-data-service
 
-Serves billing data from {productname} for the **CSP-billing-adapter**.
+SThis service serves billing data from {productname} for the **CSP-billing-adapter**.
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg-support.adoc:79]:
```suggestion
This service serves billing data from {productname} for the **CSP-billing-adapter**.
```

File: modules/specialized-guides/pages/public-cloud-guide/payg-support.adoc [hunk 3]
```diff
@@ -130,4 +145,4 @@ Failed checks are logged in:
 
 Consequences::
 . No more actions will be executed.
-. **Reposync** will stop syncing channels.
+. **reposync** will stop syncing channels.
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg-support.adoc:148]:
```suggestion
. **reposync** will stop synchronizing channels.
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/payg-limitations.adoc (hunk 1 above)
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/payg-limitations.adoc:3]:
```suggestion
== Onboarded {byos} Instances without SCC Credentials
```

**Pull request merged**